src/cookbook/use-multi-signature-transactions.md

```diff
@@ -0,0 +1,197 @@
+      ),
+    ),
+  ),
+  datamodel.IdBox('AccountId', MAD_HATTER),
+)
+
+// register Mad Hatter with the admin account
+
+await blocks.wait(async () => {
+  await clientAdmin.submitExecutable(
+    pre,
+    sugar.executable.instructions(registerAccount),
+  )
+})
+
+// Register the asset definition with the Mad Hatter's account
+await blocks.wait(async () => {
+  await Torii.submit(
+    pre,
+    pipe(
+      sugar.executable.instructions([
+        registerAssetDefinition,
+        setSignatureCondition,
+      ]),
+      (executable) =>
+        makeTransactionPayload({ executable, accountId: MAD_HATTER }),
+      (x) => makeSignedTransaction(x, signer1),
+    ),
+  )
+})
+```
+
+:::
+
+Then we build the transaction with the account's primary key:
+
+::: code-group
+
+```rust [Rust]
+let quantity: u32 = 200;
+let asset_id = AssetId::new(asset_definition_id, alice_id.clone());
+let mint_asset = MintExpr::new(quantity.to_value(), IdBox::AssetId(asset_id.clone()));
+
+let (public_key1, private_key1) = alice_key_pair.into();
+client_configuration.account_id = alice_id.clone();
+client_configuration.public_key = public_key1;
+client_configuration.private_key = private_key1;
+let client = Client::new(&client_configuration)?;
+let instructions = [mint_asset.clone()];
+let transaction = client.build_transaction(instructions, UnlimitedMetadata::new())?;
+client.submit_transaction(&client.sign_transaction(transaction)?)?;
+```
+
+```ts [TypeScript]
+const quantity = 100
+
+const tx1 = makeSignedTransaction(
+  makeTransactionPayload({
+    executable: pipe(
+      sugar.instruction.mint(
+        sugar.value.numericU32(quantity),
+        datamodel.IdBox(
+          'AssetId',
+          sugar.assetId(MAD_HATTER, CASOMILE_DEFINITION_ID),
+        ),
+      ),
+      sugar.executable.instructions,
+    ),
+    accountId: MAD_HATTER,
+  }),
+  signer1,
+)
+
+await Torii.submit(pre, tx1)
+```
+
+:::
+
+After that, we sign the same transaction with the second key and submit it:
+
+::: code-group
+
+```rust [Rust]
+let (public_key2, private_key2) = key_pair_2.into();
+client_configuration.public_key = public_key2;
+client_configuration.private_key = private_key2;
+let client_2 = Client::new(&client_configuration)?;
+let instructions = [mint_asset];
+let transaction = client_2.build_transaction(instructions, UnlimitedMetadata::new())?;
+let transaction = client_2
+    .get_original_transaction(&transaction, 3, Duration::from_millis(100))?
+    .expect("Found no pending transaction for this account.");
+client_2.submit_transaction(&client_2.sign_transaction(transaction)?)?;
+```
+
+```ts [TypeScript]
+const tx2 =
+  // we use `produce` from `immer` library
+  // it allows us to produce a new value from `tx1` without touching it in a declarative way
+  produce(tx1, (draft) => signTransaction(draft, signer2))
+
+await Torii.submit(pre, tx2)
+```
+
+:::
+
+You could find full code here: [Rust](https://www.example.com/),
+[TypeScript](https://www.example.com/).
```

Comment:
this will be links to the examples repo, right?

```suggestion
## Find More

You could find full code here: [Rust](https://www.example.com/),
[TypeScript](https://www.example.com/).
```

I think this section might also include other useful links, e.g. the link to the guide section of the docs where the topic at hand is discussed in details.

Reply:
> this will be links to the examples repo, right?

Yes